.projenrc.ts

```diff
@@ -17,7 +18,7 @@ const project = new AwsCdkApp({
         },
     },
     description: "Example of using LLRT (Low Latency Runtime) on AWS Lambda.",
-    devDeps: ["@nikovirtala/projen-aws-cdk-app"],
+    devDeps: ["@nikovirtala/projen-constructs@^0.2.26"],
```

Comment:
no need to specify version here. you can remove it:

```suggestion
    devDeps: ["@nikovirtala/projen-constructs"],
```